client/src/components/dashboard/Files.js

```diff
@@ -39,40 +43,34 @@ function Files() {
   useEffect(() => {
     setFeedbacks(allFeedbacks);
   }, [allFeedbacks]);
+
   return (
     <>
       <section>
-        <section>
-          {feedbacks.length !== 0 &&
-            feedbacks[0].feedback_id !== null &&
-            feedbacks.map((feedback) => (
-              <div>
-                <div key={feedback.feedback_id}>{feedback.feedback_text}</div>
-                <button onClick={() => deleteFeedback(feedback.feedback_id)}>
-                  delete feedback
-                </button>
-              </div>
-            ))}
-        </section>
         <div className="icon-heading">
-          <h1>Files</h1>
+          <h2>Files</h2>
           <img alt="files icon" src={filesIcon} />
```

Comment:
is "files icon" a useful description for a user interacting with this interface without seeing it? What does files icon bring to the user experience in this context?

https://supercooldesign.co.uk/blog/how-to-write-good-alt-text